engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/videoSettings/VideoSettingsScreen.java

```diff
@@ -64,26 +65,56 @@ public VideoSettingsScreen() {
     @Override
     @SuppressWarnings("unchecked")
     public void initialise() {
+
         UIDropdown<Preset> videoQuality = find("graphicsPreset", UIDropdown.class);
         if (videoQuality != null) {
+            videoQuality.setOptionRenderer(new StringTextRenderer<Preset>() {
+                @Override
+                public String getString(Preset value) {
+                    return translationSystem.translate("${engine:menu#" + value.getDisplayName() + "}");
+                }
+            });
             videoQuality.setOptions(Lists.newArrayList(Preset.CUSTOM, Preset.MINIMAL, Preset.NICE, Preset.EPIC, Preset.INSANE, Preset.UBER));
             videoQuality.bindSelection(new PresetBinding(config.getRendering()));
         }
 
         UIDropdown<ViewDistance> viewDistance = find("viewDistance", UIDropdown.class);
         if (viewDistance != null) {
+            viewDistance.setOptionRenderer(new StringTextRenderer<ViewDistance>() {
+                @Override
+                public String getString(ViewDistance value) {
+                    return translationSystem.translate("${engine:menu#" + value.getDisplayName() + "}");
+                }
+            });
             viewDistance.setOptions(Arrays.asList(ViewDistance.values()));
             viewDistance.bindSelection(BindHelper.bindBeanProperty("viewDistance", config.getRendering(), ViewDistance.class));
         }
 
         UIDropdown<WaterReflection> waterReflection = find("reflections", UIDropdown.class);
         if (waterReflection != null) {
+            waterReflection.setOptionRenderer(new StringTextRenderer<WaterReflection>() {
+                @Override
+                public String getString(WaterReflection value) {
+                    return translationSystem.translate("${engine:menu#" + value.getDisplayName() + "}");
+                }
+            });
             waterReflection.setOptions(Lists.newArrayList(WaterReflection.SKY, WaterReflection.GLOBAL, WaterReflection.LOCAL));
             waterReflection.bindSelection(new WaterReflectionBinding(config.getRendering()));
         }
 
         UIDropdown<ScreenshotSize> screenshotSize = find("screenshotSize", UIDropdown.class);
         if (screenshotSize != null) {
+            screenshotSize.setOptionRenderer(new StringTextRenderer<ScreenshotSize>() {
+                @Override
+                public String getString(ScreenshotSize value) {
+                    if ( value.getDisplayName().equals("1080p") || value.getDisplayName().equals("720p")) {
```

Comment:
Please avoid special cases, when possible. You "translate" `1080p` to `1080p` in the root language definition file (menu.lang). Then, it can treated like the other values, but will remain constant as no real translation file has a translation for it.